client/state/plans/interval-types.jsx

```diff
@@ -0,0 +1,17 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+
+import { TERM_ANNUALLY, TERM_BIENNIALLY, TERM_MONTHLY } from '../../lib/plans/constants';
+
+export const TERM_TO_INTERVAL_TYPE = {
+	[ TERM_MONTHLY ]: 'monthly',
+	[ TERM_ANNUALLY ]: 'yearly',
+	[ TERM_BIENNIALLY ]: '2yearly',
+};
```

Comment:
The values here are essentially interval slugs that we'd expect to find as part of a url?

If I'm not mistaken, the trend seems to be replacing the constants and object mappings with functions. That seems good, and is nicer for the ability to test. Do you think a function should be exported here, as opposed to the object, in that same spirit? It would be very simple to test.

A function (and some test) would also lead to the question of the `undefined` that's lurking here. This leads to a potential `undefined` returned from `getIntervalTypeFromCurrentPlan`, where other cases are returning `null`.